awswrangler/distributed/_distributed.py

```diff
@@ -115,6 +119,9 @@ def initialize_ray(
                 "object_store_memory": object_store_memory,
                 "_redis_password": redis_password,
                 "_memory": object_store_memory,
+                "runtime_env": {
```

Comment:
Setting this in tox as an environment variable does not get to to `ray_init_kwargs.runtime_env` from what I can tell. So I added this logic to achieve the same.